Automattic/woocommerce-payments
# Transaction Details → Show resolution status footer for inquiries

Fixes #7191 

#### Changes proposed in this Pull Request

This PR adds dispute information to the Transaction Details screen for closed or under review **inquiries**, with a link to the submitted evidence if available.


**Inquiry: Under review**
![image](https://github.com/Automattic/woocommerce-payments/assets/3147296/3e10e1c9-4f08-4162-bdd6-5bd77ca88c27)

**Inquiry: Closed** (challenged)
![image](https://github.com/Automattic/woocommerce-payments/assets/3147296/d6b6c0c8-c53d-4cd0-bc90-2e99f3c3e20d)

**Inquiry: Closed** (accepted)
![image](https://github.com/Automattic/woocommerce-payments/assets/3147296/1e2739d2-2f5e-4921-93d0-de7b4fee5fc4)

**Mobile viewport**
<img width="375" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/090d750c-22a0-46cc-a4a1-f9485880f44d">


#### Testing instructions






* Enable feature flag by running `update_option( '_wcpay_feature_dispute_on_transaction_page', '1' );` in [WP Console](https://wordpress.org/plugins/wp-console/) or by modifying your database table `wp_options` directly.

* **Inquiry: Under review**
	* Create a disputed inquiry transaction using the card `4000000000001976` at checkout.
	* Find the inquiry in Payments → Disputes and click it to view the dispute details screen.
	* Click Challenge and submit the challenge form with arbitrary evidence. This will change the dispute's status from `warning_needs_response` to `warning_under_review`.
	* Navigate to the transaction details screen for this dispute.
	* Notice the dispute resolution footer that matches the screenshot above.
* **Inquiry: Closed** (challenged)
	* Create a disputed inquiry transaction using the card `4000000000001976` at checkout.
	* Find the inquiry in Payments → Disputes and click it to view the dispute details screen.
	* Click Challenge and submit the challenge form **with `winning_evidence` in the “Additional details” field**. This will change the dispute's status from `warning_needs_response` to `warning_closed`.
	* Navigate to the transaction details screen for this dispute.
	* Notice the dispute resolution footer that matches the screenshot above.
* **Inquiry: Closed** (accepted)
	* Create a disputed inquiry transaction using the card `4000000000001976` at checkout.
	* Find the inquiry in Payments → Disputes and click it to view the dispute details screen.
	* Click Accept. This will change the dispute's status from `warning_needs_response` to `warning_closed`.
	* Navigate to the transaction details screen for this dispute.
	* Notice the dispute resolution footer that matches the screenshot above.


-------------------


**Post merge**
